wp-content/plugins/tw-programs/tw-programs.php

```diff
@@ -44,6 +44,7 @@ function tw_programs_taxonomy() {
 		'show_in_nav_menus' => false,
 		'show_tagcloud'     => false,
 		'show_in_rest'      => true,
+		'rewrite'           => array( 'slug' => 'program' ),
```

Comment:
This will conflict with changes coming in the `feat/102-the-world-theme` branch. Since the slug portions of permalink URL's from WP can be used as ID's, we are refactoring the frontend to have matching routes so canonical URL's won't need to do the extra alias lookup request. We ended up with all permalink prefixes being plurals of the resource type name.

Reply:
It should work without that rewrite as well so i removed it.